pages/contributors.tsx

```diff
@@ -7,41 +7,64 @@ import { Spinner } from "react-bootstrap"
 import Lottie from "react-lottie"
 import Card from "../components/Cards/Card"
 
-const defaultOptions = {
+type ContributorSerializer = {
+    handle_name:string
+    name:string
+    handle_url:string
+    image:string
+    github:string
+    description:string[]
+}
+
+type ContriResponseSerializer = {
```

Comment:
If I am not wrong, is this the type for the Card component props?

If yes, we can reuse the same type from PR #108. Maybe it's better if we store the types in the `types` folder and import them wherever required. What do you think? @tiger-yash @Prakhar-Agarwal-byte

Reply:
`ContributorSerializer` is kind of common in both the files, but it is used only in these 2 files as it is basically produced specifically for contributors page using responses from github APIs.
It highly unlikely that we use it again, so we may also leave them be. I am open to other suggestions, though.